js/index.less

```diff
@@ -128,29 +153,78 @@ body {
                 margin-right: -20px;
                 padding-left: 20px;
                 padding-right: 20px;
+
+                .fa {
+                  opacity: 1;
+                }
             }
         }
+
+        .fa {
+          margin-left: 10px;
+          opacity: 0;
+        }
     }
 }
 
 .StyleguideContent {
-    background: linear-gradient(-45deg,#f1f1f1,#f1f1f1 25%,#e9e9e9 25%,#e9e9e9 50%,#f1f1f1 50%,#f1f1f1 75%,#e9e9e9 75%);
-    background-size: 50px 50px;
-    bottom: 20px;
-    left: 270px;
-    position: fixed;
-    right: 20px;
-    top: 90px;
-
-    body[data-iframed] & {
-        top: 20px;
-    }
+  position: fixed;
+  bottom: 20px;
+  left: 270px;
+  right: 20px;
+  top: 90px;
+
+  body[data-iframed] & {
+      top: 20px;
+  }
 }
 
 .StyleguideExample {
-    border: none;
+    border: 0;
     display: block;
-    height: ~'calc(100% - 40px)';
+    height: ~'calc(100% - 87px)';
     margin: 20px;
     width: ~'calc(100% - 40px)';
 }
+
+.StyleguideTabs {
+  margin: 0;
+  background-color: #ededed;
+
+  li {
+    display: inline-block;
+
+    &[data-active] a {
+      color: #262626;
+      background-color: white;
+      cursor: auto;
+
+      &:before {
+        opacity: 1;
+        top: 0;
+      }
+    }
+  }
+
+  > li a {
```

Comment:
Update these element selectors to be class based along with change to template mentioned previously